core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -106,7 +107,16 @@ object Boot extends App with LazyLogging {
     val jsonFactory = JacksonFactory.getDefaultInstance
     val clientSecrets = GoogleClientSecrets.load(jsonFactory, new StringReader(gcsConfig.getString("secrets")))
     val clientEmail = gcsConfig.getString("serviceClientEmail")
+
+    val defaultServiceAccountPem = GoogleCredentialModes.Pem(WorkbenchEmail(clientEmail), new File(gcsConfig.getString("pathToPem")), Option(WorkbenchEmail(gcsConfig.getString("subEmail"))))
+
+    val groupServiceAccountPool: Array[GoogleCredentialMode] = try { gcsConfig.getStringList("groupServiceAccountPool").map(GoogleCredentialModes.Json).toArray } catch {
```

Comment:
not sure of a more elegant way to do this. try catch was the most straightforward way i could think of to parse an optional string list